### ISSUE TITLE
User icon is too large compared to neighbor icons

### ISSUE THREAD

**Issue by SeaDve** [rafaelmardojai/firefox-gnome-theme#414]
![image](https://user-images.githubusercontent.com/64297935/177900640-ca1ed1c3-4d48-4654-8678-7ed4442d89de.png)

**Comment by rafaelmardojai** [comment]
Yeah, I increased it to match other GNOME apps with similar buttons, like Fractal. See #411.

**Comment by rafaelmardojai** [comment]
Honestly I don't use this button, maybe moving it all to the left makes more sense? 😅

**Comment by SeaDve** [comment]
> Yeah, I increased it to match other GNOME apps with similar buttons, like Fractal. See #411.

Ah, that makes sense too. 

> Honestly I don't use this button, maybe moving it all to the left makes more sense? 😅

Yeah, it looks better on the farthest left.

I think I'll close this then. Thanks!
